examples/kitchen-sink/README.md

```diff
@@ -0,0 +1,7 @@
+# Kitchen Sink
```

Comment:
Loving this so much. Have we discussed alternative names for this example? Not sure if everyone will automatically associated kitchen-sink with an sdk playground.